vulnerable_people_form/form_pages/check_your_answers.py

```diff
@@ -7,7 +7,7 @@
 from .shared.routing import route_to_next_form_page, dynamic_back_url
 from .shared.session import persist_answers_from_session, form_answers, get_summary_rows_from_form_answers, \
     get_postcode_tier, is_very_high_plus_shielding_without_basic_care_needs_answer
-from ..integrations import govuk_notify_client, spl_check
+from ..integrations import govuk_notify_client, spl_check, notifications
```

Comment:
if we are not using the notify client i would remove it from the code base along with the templates